### PR TITLE
figure 4 results edits

### PR DESCRIPTION
**Pull request by jclagarde**
Made some texts edits addressing the comments for figure 4 results and changed the transparency of the data points from 0.33 to 0.5. Please let me know if this is okay or if there should be any further changes. Thank you!

### CODE REVIEW

File: manuscript/03.results.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ To evaluate how accurately a tHMM model could infer the behavior of multi-state
...
...creasing breadth and depth.** (a) Two-state populations of increasing breadth (increasing number of initial cells and therefore lineages) and of increasing depth (increasing experiment time) are analyzed. The states are shown in green and blue colors and red shows cell death. Each point in the scatter plots represents the estimated value for a lineage with the number of cells shown in the x-axis. The solid lines are the Lowess lines for the scatter points. The accuracy in estimating the Bernoulli parameters, which imply cell death or cell division, for G1 and S/G2 phase, respectively, is shown in (b-c). The distance between the true and estimated Gamma distributions associated with phase lengths for the two states is depicted in (d). This is measured using the Wasserstein distance. The model performance shown is based on state assignment accuracy (e), the error between the estimated and the true transition rate matrix (f), and the initial probability vector (g). Note that the Wasserstein distance between the true and estimated distributions for each state is much lower than the distance for two distributions that are quite similar (Fig. {@fig:wass}b).](figure5.svg){#fig:performSyn}
```
Comment (Farnazmdi) [inline manuscript/03.results.md:39]: The explanations you added should go to the main text, not the caption.

Comment (jclagarde) [reply manuscript/03.results.md]: Thank you for letting me know! I just made some changes in my most recent branch. Please let me know what you think. I had also repositioned the cartoon for Supplementary Figure 9 and 10 to be a bit more centered

Comment (Farnazmdi) [reply manuscript/03.results.md]: Thanks! Sounds good to me. Please ask aarmey for merge.

**Review by Farnazmdi — APPROVED** [review]

**Comment by jclagarde** [comment]
@aarmey Hi Dr. Meyer, when you get the chance can you please review the edits I made and if it is okay, please merge the pull request. Thank you!

**Comment by aarmey** [comment]
Looks good—just made one edit. Will merge now.

**Pull request merged**